reference/5.1/Microsoft.PowerShell.Security/Protect-CmsMessage.md

```diff
@@ -79,24 +83,24 @@ ValidityPeriodUnits = "1000"
 PS C:\> Certreq -new DocumentEncryption.inf DocumentEncryption.cer
 ```
 
-Before you can run the **Protect-CmsMessage** cmdlet, you must create an encryption certificate.
-Using the following text, change the name in the Subject line to your name, email, or other identifier, and save the certificate in a file (such as DocumentEncryption.inf, as shown in this example).
-
 ### Example 2: Encrypt a message sent by email
-```
-PS C:\> $Protected = "Hello World" | Protect-CmsMessage -To "*youralias@emailaddress.com*"
-```
 
 In the following example, you encrypt a message, Hello World, by saving the message in a variable, and then piping it to the **Protect-CmsMessage** cmdlet.
+
 The *To* parameter uses the value of the Subject line in the certificate.
 
-### Example 3: View document encryption certificates
+```PowerShell
+PS C:\> $Protected = "Hello World" | Protect-CmsMessage -To "*youralias@emailaddress.com*"
```

Comment:
Remove the prompt strings from the examples. See https://github.com/PowerShell/PowerShell-Docs/blob/staging/contributing/FORMATTING-CODE.md#general-guidelines